### PR TITLE
CGT-1499: lossedBroughtFowardValue added.

### PR DESCRIPTION
**Pull request by MJCallahanPage**

### CODE REVIEW

File: app/controllers/resident/shares/DeductionsController.scala [hunk 1]
```diff
@@ -55,12 +46,122 @@ trait DeductionsController extends FeatureLock {
...
+    lossesBroughtForwardValueForm.bindFromRequest.fold(
+      errors => { for {
+        disposalDate <- getDisposalDate
+        disposalDateString <- formatDisposalDate(disposalDate.get)
+        taxYear <- calcConnector.getTaxYear(disposalDateString)
+      } yield {
+        BadRequest(commonViews.lossesBroughtForwardValue(
+          errors,
+          taxYear.get,
+          navBackLink = routes.DeductionsController.lossesBroughtForward().url,
```
Comment (jameshforster) [inline app/controllers/resident/shares/DeductionsController.scala]: Need to be converted to private vals

File: app/controllers/resident/shares/DeductionsController.scala [hunk 2]
```diff
@@ -55,12 +46,122 @@ trait DeductionsController extends FeatureLock {
...
+        case Some(data) => lossesBroughtForwardValueForm.fill(data)
+        case _ => lossesBroughtForwardValueForm
+      }
+    }
 
+    def routeRequest(taxYear: TaxYearModel, formData: Form[LossesBroughtForwardValueModel]): Future[Result] = {
+      Future.successful(Ok(commonViews.lossesBroughtForwardValue(
+        formData,
+        taxYear,
+        navBackLink = routes.DeductionsController.lossesBroughtForward().url,
```
Comment (jameshforster) [inline app/controllers/resident/shares/DeductionsController.scala]: Needs to be converted to private vals

**Comment by jameshforster** [comment]
Please fix the conflicts

**Comment by MJCallahanPage** [comment]
Merge conflicts fixed

**Pull request merged**